File: mario/mario-more/mario.c
// Creates a pyramid of blocks for mario game design

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asks the user for height of blocks
    int blocks;
    do
    {
        blocks = get_int("How many blocks high?  ");
    }
    while (blocks < 1 || blocks > 8);

    // Main Loop that runs the code inside the amount of blocks the user input (height)
    int c;
    for (c = 0; c < blocks; c++)
    {
        int i;
        int j;
        int s;

        // Loop that prints the initial spacing before a first block on a line
        for (s = c; s < (blocks - 1) ; s++)
        {
            printf(" ");
        }

        // Prints the first set of blocks on a line
        for (i = 0; i <= c; i++)
        {
            printf("#");
        }

        // Prints a gap between our 2 sets of blocks
        printf("  ");

        // Prints the second set of blocks on a line
        for (j = 0; j <= c; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# Creates a reflection of blocks x high
def main():
    height = get_height()
    for c in range(height):
        for s in range(c, height - 1):
            print(" ", end='')
        for i in range(c + 1):
            print("#", end='')
        print("  ", end='')
        for j in range(c + 1):
            print("#", end='')
        print()


# Asks user to set a height
def get_height():
    while True:
        blocks = input("How many blocks high? ")
        if blocks.isdigit():
            blocks = int(blocks)
            if blocks >= 1 and blocks <= 8:
                return blocks


# Call main
main()
